community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,420 @@
+.. _doc_pr_review_guidelines:
+
+Pull request review process
+===========================
+
+.. note::
+
+    This page is intended to provide insight into the pull request (PR) review
+    process that we aspire to. As such, it is primarily targeted at engine
+    maintainers who are responsible for reviewing and approving pull requests.
+    That being said, much of the content is useful for prospective contributors
+    wanting to know how to ensure that their PR is merged.
+
+From a high level, the ideal life cycle of a pull request looks like the
+following: 
+
+  1. A contributor opens a PR that fixes a specific problem (GitHub `issue
+     <https://github.com/godotengine/godot>`_ or implementing a `proposal
+     <https://github.com/godotengine/godot-proposals>`_).
```

Comment:
```suggestion
  1. A contributor opens a PR that fixes a specific problem (optimally closing
    a GitHub `issue <https://github.com/godotengine/godot>`_ or implementing
    a `proposal <https://github.com/godotengine/godot-proposals>`_).
```

I think you've lost a couple of words here, "optimally closing a", during editing.